exercicios/para-casa/exercicio16.py

```diff
@@ -0,0 +1,18 @@
+#LOJA DE TINTAS:
+
+tamanho = float(input("QUANTOS METROS QUADRADOS TEM A ÁREA A SER PINTADA? "))
+
+metroQuadradoPorLitro = 3
+
+litrosNecessarios = tamanho / metroQuadradoPorLitro
+
+lataLitros = 18
+lataPreco = 80
+
+latasNecessarias = litrosNecessarios / lataLitros

```

Comment:
Quase. Mas há um porém: Se o resultado da divisão não for exato, por exemplo 35/18, o resultado será 1 e faltará tinta.